packages/gatsby-plugin-sharp/README.md

```diff
@@ -95,19 +95,17 @@ a base64 image to use as a placeholder) you need to implement the "blur up"
 technique popularized by Medium and Facebook (and also available as a Gatsby
 plugin for Markdown content as gatsby-remark-images).
 
-When both a `maxWidth` and `maxHeight` are provided, sharp will use `COVER` as a fit strategy by default. This might not be ideal so you can now choose between `COVER`, `CONTAIN` and `FILL` as a fit strategy. To see them in action the [CSS property object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) comes close to its implementation.
-
-#### Note
-
-fit strategies `CONTAIN` and `FILL` will not work when `cropFocus` is assigned to [sharp.strategy][6]. The `cropFocus` option cannot be `ENTROPY` or `ATTENTION`
+When both a `maxWidth` and `maxHeight` are provided, sharp will [resize the image][6] using
+`COVER` as a fit strategy by default. You can choose between `COVER`, `CONTAIN`, `FILL`,
```

Comment:
Should this be `CONTAIN`?

Reply:
The default is `COVER` for gatsby-plugin-sharp. I changed the default for gatsby-image to `contain` so that gatsby-plugin-sharp "owns" the size and shape of the image, rather than having gatsby-image perform a _second_ fit after after gatsby-plugin-sharp.